resolwe_bio/processes/expression/slamdunk.py

```diff
@@ -0,0 +1,105 @@
+import os
+
+from plumbum import TEE
+
+from resolwe.process import Process, Cmd, DataField, BooleanField, IntegerField, FileField, DirField
+
+
+class SlamdunkAllPaired(Process):
+    """Run slamdunk for paired-end reads with alignment to transcriptome."""
+
+    slug = 'slamdunk-all-paired'
+    process_type = 'data:reads:fastq:paired'
+    name = 'Slamdunk(paired-end)'
+    requirements = {
+        'expression-engine': 'jinja',
+        'executor': {
+            'docker': {
+                'image' : 'resolwebio/slamdunk:1.0.0'
+            },
+        },
+        'resources': {
+            'cores': 8,
+            'memory': 16384,
+        },
+    }
+    data_name = '{{ reads|sample_name|default("?") }}'
+    version = '1.0.0'
+    process_type = 'data:expression:slamdunk'
+
+
+    class Input:
+        """Input fields for SlamdunkAllPaired."""
+
+        reads = DataField('reads:fastq:paired', label='Reads')
+        transcriptome = DataField('genome:fasta', label='FASTA file containig transcripts')
```

Comment:
Rather use input type `data:seq:nucleotide`. Data type `genome:fasta` also works, but the process that prepares this type of Data object also computes indices for aligner tools (except for STAR aligner).